pages/features.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import FeatureHeaderSection from '../components/Features/FeatureHeaderSection';
+import FeatureTwoImage from '../components/Features/FeatureTwoImage';
+import FeatureVideoText from '../components/Features/FeatureVideoText';
+import FeatureFormSection from '../components/Features/FeatureFormSection';
+import AppLayout from '../components/AppLayout';
+
+export default function Features() {
+  const images = [
+    'https://givngo.io/wp-content/themes/givngo/images/OOOTTTIIIIII.png',
+    'https://givngo.io/wp-content/themes/givngo/images/2305-6-Eblast-One-Tree-Planted-One-Tree-Planted-2.jpg',
+  ];
+  const image1 = [
+    'https://givngo.io/wp-content/themes/givngo/images/No-Color-Present-Tense.jpg',
+    'https://givngo.io/wp-content/themes/givngo/images/Screen-Shot-2019-11-15-at-11.22.20.png',
+  ];
+  return (
+    <AppLayout>
+      <FeatureHeaderSection />
+      <FeatureTwoImage
+        urls={images}
+        position="right"
+        srcsetParam="https://givngo.io/wp-content/themes/givngo/images/OOOTTTIIIIII-p-500.png 500w, https://givngo.io/wp-content/themes/givngo/images/OOOTTTIIIIII-p-800.png 800w, https://givngo.io/wp-content/themes/givngo/images/OOOTTTIIIIII-p-1080.png 1080w, https://givngo.io/wp-content/themes/givngo/images/OOOTTTIIIIII.png 2560w"
```

Comment:
Instead of passing in these images as params, we should probably host the images on the client side. That way we don't need this parameter and all the different image sizes.